packages/teams-js/src/public/liveShareHost.ts

```diff
@@ -4,149 +4,184 @@ import { FrameContexts } from './constants';
 import { runtime } from './runtime';
 
 /**
- * @hidden
- * Allowed roles during a meeting.
+ * APIs involving Live Share, a framework for building real-time collaborative apps.
+ * For more information, visit https://aka.ms/teamsliveshare
  *
- * @beta
+ * @see LiveShareHost
  */
-export enum UserMeetingRole {
-  guest = 'Guest',
-  attendee = 'Attendee',
-  presenter = 'Presenter',
-  organizer = 'Organizer',
-}
-
-/**
- * @hidden
- * State of the current Live Share sessions backing fluid container.
- *
- * @beta
- */
-export enum ContainerState {
+export namespace liveShare {
   /**
-   * The call to `LiveShareHost.setContainerId()` successfully created the container mapping
-   * for the current Live Share session.
+   * @hidden
```

Comment:
Why is this hidden? Adding this tag means dev docs won't be created for this enum, which would be pretty unusual for a non-beta enum that is used in exported and documented functions.

Reply:
I think I just have that question for all @hidden documentation in this file. Why don't you want public documentation for them?

Reply:
Developers don’t use those APIs directly, they are used in a very specific workflow within the Live Share SDK and have little utility outside of it. We thought about making them public but it’s not something anybody would know how to do, so we figured it’d be more confusing than not to document it.